spec/workers/cleanup_expired_access_tokens_worker_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+describe CleanupExpiredAccessTokensWorker do
```

Comment:
Should we be testing the behaviour of Doorkeeper::AccessCleanup here? The worker doesn't actually do any of this stuff, all it does is call another method.